hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/validation/StandardizedAttributeValidator.java

```diff
@@ -136,4 +137,10 @@ private static boolean contains(byte[] a, byte val) {
         }
         return false;
     }
+
+    @Override
+    public boolean isImmutableKey(Key key) {
```

Comment:
Is this intended to be,
```
public boolean isImmutableKey(@NonNull final Key key) {
        return requireNonNull(key).hasKeyList() && key.keyList().isEmpty();
}
```
As written seems to have the null check in the wrong place; and `keysOrElse()` is redundant if `hasKeyList()` is true.

Reply:
Added requireNonNull to remove sonar warning: Method invocation 'keysOrElse may produce 'NullPointerException'.
Same for keysOrElse(emptyList()). key.keyList().isEmpty() is not valid for the hapi.node.base.Key